common/lib/capa/capa/inputtypes.py

```diff
@@ -85,6 +85,7 @@ def __init__(self, status, gettext_func=unicode):
         names = {
             'correct': _('correct'),
             'incorrect': _('incorrect'),
+            'partially-correct': _('partially-correct'),
```

Comment:
It's probably not. I thought it was turning problem status into a CSS class name, but there's something I don't understand here, because changing the right-hand text doesn't change the name of the CSS class. I'll keep poking at this intermittently.